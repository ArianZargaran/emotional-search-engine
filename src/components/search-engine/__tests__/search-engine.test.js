import React from "react";
import SearchEngine from "../search-engine";

it('renders SearchEngine correctly', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<SearchEngine />);
  const tree = renderer.getRenderOutput();
  expect(tree).toMatchSnapshot();
});