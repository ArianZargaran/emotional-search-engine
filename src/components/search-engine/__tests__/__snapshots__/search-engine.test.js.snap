// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SearchEngine correctly 1`] = `
<div
  className="ese-search-engine_wrapper"
>
  <div
    className="ese-search-engine"
  >
    <SearchFormWrapper />
  </div>
</div>
`;
